src/react/types/types.ts

```diff
@@ -151,11 +158,18 @@ export interface BackgroundQueryHookOptions<
   | 'canonizeResults'
   | 'returnPartialData'
   | 'refetchWritePolicy'
-  | 'skip'
 > {
   fetchPolicy?: BackgroundQueryHookFetchPolicy;
   suspenseCache?: SuspenseCache;
   queryKey?: string | number | any[];
+
+  /**
+   * If `true`, the query is not executed. The default value is `false`.
+   *
+   * @deprecated We recommend using skipToken in place of the skip option. This 
+   * is more type-safe in most cases.
```

Comment:
```suggestion
```suggestion
   * @deprecated We recommend using `skipToken` in place of the skip option. This
   * is more type-safe in most cases.
   * @example recommended usage of `skipToken`:
   * ```js
   * import { skipToken, useBackgroundQuery } from '@apollo/client';
   *
   * const { data } = useBackgroundQuery(query, id ? { variables: { id } } : skipToken)
   * ```
   */
```
```